### PR TITLE
add content for clinic us

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/clinic-user-guide.md [hunk 1]
```diff
@@ -140,19 +144,21 @@ Access Token（以下简称为 Token）用于 Diag 上传数据时的用户认
 
 ```shell
 # namespace：和 TiDB Operator 处于同一 namespace 中
-# diag.clinicToken：请在 "https://clinic.pingcap.com.cn" 中登录并获取您的 Token。
-helm install --namespace tidb-admin diag-collector pingcap/diag --version v0.7.1 \
+# diag.clinicToken：请在 "https://clinic.pingcap.com.cn" 或 "https://clinic.pingcap.com" 中登录并获取您的 Token。
+helm install --namespace tidb-admin diag-collector pingcap/diag --version v0.9.0 \
         --set diag.clinicToken=${clinic_token}
+        --set diag.clinicRegion= ${clinic_region}  # CN or US
```
Comment (nexustar) [inline zh/clinic-user-guide.md:150]: no space here

File: zh/clinic-user-guide.md [hunk 2]
```diff
@@ -140,19 +144,21 @@ Access Token（以下简称为 Token）用于 Diag 上传数据时的用户认
...
 > **注意：**
 >
 > 如果访问 Docker Hub 网速较慢，可以使用阿里云上的镜像：
 >
 > ```shell
-> helm install --namespace tidb-admin diag-collector pingcap/diag --version v0.7.1 \
+> helm install --namespace tidb-admin diag-collector pingcap/diag --version v0.9.0 \
 >     --set image.diagImage=registry.cn-beijing.aliyuncs.com/tidb/diag \
 >     --set diag.clinicToken= ${clinic_token}
+      --set diag.clinicRegion= ${clinic_region}
```
Comment (nexustar) [inline zh/clinic-user-guide.md:161]: no space here

File: zh/clinic-user-guide.md [hunk 3]
```diff
@@ -348,27 +354,30 @@ Make sure diag-collector components are running:
...
     - 如果集群未开启 TLS，可以设置 `diag.tlsEnabled=false`，此时创建的 Role 将不会带有 `secrets` 的 `get` 和 `list` 权限。
 
         ```shell
-        helm install --namespace tidb-cluster diag-collector pingcap/diag --version v0.7.1 \
+        helm install --namespace tidb-cluster diag-collector pingcap/diag --version v0.9.0 \
             --set diag.clinicToken=${clinic_token} \
             --set diag.tlsEnabled=false \
-            --set diag.clusterRoleEnabled=false
+            --set diag.clusterRoleEnabled=false  \
+            --set diag.clinicRegion= ${clinic_region}
```
Comment (nexustar) [inline zh/clinic-user-guide.md:370]: no space here

File: zh/clinic-user-guide.md [hunk 4]
```diff
@@ -348,27 +354,30 @@ Make sure diag-collector components are running:
...
     - 如果访问 Docker Hub 网速较慢，可以使用阿里云上的镜像：
 
         ```shell
-        helm install --namespace tidb-cluster diag-collector pingcap/diag --version v0.7.1 \
+        helm install --namespace tidb-cluster diag-collector pingcap/diag --version v0.9.0 \
             --set image.diagImage=registry.cn-beijing.aliyuncs.com/tidb/diag \
             --set diag.clinicToken= ${clinic_token} \
-            --set diag.clusterRoleEnabled=false
+            --set diag.clusterRoleEnabled=false \
+            --set diag.clinicRegion= ${clinic_region}
```
Comment (nexustar) [inline zh/clinic-user-guide.md:380]: no space here

File: en/clinic-user-guide.md [hunk 5]
```diff
@@ -140,9 +141,10 @@ Deploy Diag using the following `helm` command and the latest Diag image is pull
 
 ```shell
 # namespace: the same as that of TiDB Operator
-# diag.clinicToken: get your token in "https://clinic.pingcap.com.cn"
-helm install --namespace tidb-admin diag-collector pingcap/diag --version v0.7.1 \
+# diag.clinicToken: get your token in "https://clinic.pingcap.com.cn" or "https://clinic.pingcap.com"
+helm install --namespace tidb-admin diag-collector pingcap/diag --version v0.9.0 \
         --set diag.clinicToken=${clinic_token}
+        --set diag.clinicRegion= ${clinic_region}  # CN or US
```
Comment (nexustar) [inline en/clinic-user-guide.md:147]: no space here

File: zh/clinic-user-guide.md [hunk 6]
```diff
@@ -348,27 +354,30 @@ Make sure diag-collector components are running:
 2. 通过如下 `helm` 命令部署 Diag，从 Docker Hub 下载最新 Diag 镜像。
 
     ```shell
-    helm install --namespace tidb-cluster diag-collector pingcap/diag --version v0.7.1 \
+    helm install --namespace tidb-cluster diag-collector pingcap/diag --version v0.9.0 \
         --set diag.clinicToken=${clinic_token} \
-        --set diag.clusterRoleEnabled=false
+        --set diag.clusterRoleEnabled=false  \
+        --set diag.clinicRegion= ${clinic_region}
```
Comment (nexustar) [inline zh/clinic-user-guide.md:360]: no space here

**Review by nexustar — APPROVED** [review]
rest LGTM

File: en/clinic-data-collection.md [hunk 7]
```diff
@@ -11,7 +11,9 @@ When you [collect data using Diag client (Diag)](clinic-user-guide.md), you can
...
 
-Clinic Server is a diagnostic service deployed in the cloud. Currently, you can upload the collected diagnostic data only to Clinic Server China. The uploaded data is stored in an Amazon S3 storage set up by PingCAP in AWS China (Beijing) Region. PingCAP strictly controls permissions for data access and only allows authorized internal technical support staff to access the uploaded data.
+Clinic Server is a diagnostic service deployed in the cloud，it has two independent backend services with different data storage localtion:
+- [Clinic Server in China](https://clinic.pingcap.com.cn)：If you upload the collected data to Clinic Server in China, the data will be stored in the S3 service in AWS China (Beijing). PingCAP has strict access control over data access rights, and only authorized PingCAP technicians can access the data.
+- [Clinic Server in US ](https://clinic.pingcap.com)：If you upload the collected data to Clinic Server in US, the data will be stored in the S3 service in AWS US region. PingCAP has strict access control over data access rights, and only authorized PingCAP technicians can access the data.
```
Comment (Oreoxmt) [inline en/clinic-data-collection.md:16]:
```suggestion
- [Clinic Server in US](https://clinic.pingcap.com)：If you upload the collected data to Clinic Server in US, the data will be stored in the S3 service in AWS US region. PingCAP has strict access control over data access rights, and only authorized PingCAP technicians can access the data.
```

File: zh/clinic-user-guide.md (hunk 4 above)
Comment (Oreoxmt) [reply zh/clinic-user-guide.md]: > no space here

done

File: en/clinic-user-guide.md [hunk 8]
```diff
@@ -104,11 +104,27 @@ Follow these steps to check the user access:
...
 ### Step 2: Log in to the Clinic Server and get an access token
 
-When Diag uploads data, the access token is used to identify the user and ensures that the data from Diag is uploaded to the organization created by the user. You need to log in to the [Clinic Server](https://clinic.pingcap.com.cn/portal/#/login) to get a token.
+When Diag uploads data, the access token is used to identify the user and ensures that the data from Diag is uploaded to the organization created by the user. You need to log in to the Clinic Server to get a token.
 
-1. Sign up and log in to the Clinic Server.
+1. Log in to the Clinic Server.
 
-    Go to [Clinic Server](https://clinic.pingcap.com.cn/portal/#/login) and select **Sign in with AskTUG** to enter the AskTUG community login page. If you do not have an AskTUG account, you can create one on that page.
+    <SimpleTab>
```
Comment (ran-huang) [inline en/clinic-user-guide.md:111]: ditto. please put US server first.

**Review by ran-huang — APPROVED** [review]
LGTM

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**